checker/raw_result.go

```diff
@@ -136,11 +136,36 @@ type VulnerabilitiesData struct {
 	Vulnerabilities []clients.Vulnerability
 }
 
+type SecurityPolicyInformationType string
+
+const (
+	// forms of security policy hints being evaluated.
+	SecurityPolicyInformationTypeEmail SecurityPolicyInformationType = "emailAddress"
+	SecurityPolicyInformationTypeLink  SecurityPolicyInformationType = "httpLink"
+	SecurityPolicyInformationTypeText  SecurityPolicyInformationType = "vulnDisclosureText"
+)
+
+type SecurityPolicyValueType struct {
+	Match      string // Snippet of match
+	LineNumber uint   // Line number in policy file of match
+	Offset     uint   // Offset in the line of the match
+}
+
+type SecurityPolicyInformation struct {
+	InformationType  SecurityPolicyInformationType
+	InformationValue SecurityPolicyValueType
+}
+
 // SecurityPolicyData contains the raw results
 // for the Security-Policy check.
 type SecurityPolicyData struct {
-	// Files contains a list of files.
-	Files []File
+	// security policy information found in repo or org
```

Comment:
why not just:
```golang
type PolicyFile {
  File File
  InformationType
  ContentLength
}
type SecurityPolicyData struct {
   PolicyFiles []PolicyFile
}
```

Reply:
I will do that, just to be specific:

```
type PolicyFile {
   File File
   Information []SecurityPolicyInformation
   ContentLength uint
}
type SecurityPolicyData struct {
   PolicyFiles []PolicyFile
}
```

Reply:
one additional comment - looking back on this new spec... for Security-Policy check, it is only looking for ONE policy file (by name), then the search finds an existing file (by name) either in the repo or the org's repo, the search stops--hence there would not be any more than one file.

therefore `PolicyFiles []PolicyFile` should/would/could be simply `PolicyFile PolicyFile` - it was that observation when I started this PR that the `SecurityPolicyData struct` is the way it is with the last commit reviewed here.

Reply:
my reasoning was to re-use the info in File (Match, LineNumber, Offset) instead of creating a new struct for it. I fully agree with you it's not optimal either due to your observation. It's probably fine to keep as-is. 

One thing we should change is allow for reporting multiple policy files. Today the code reports only one policy file, but if we decide to change it in the future we don't want to break this structure. So we just need to make a list of *some-struct* with the struct being the fields you currently have in `SecurityPolicyData`.